scripts/ci/travis.sh

```diff
@@ -67,35 +67,24 @@ done
 
 ./cardano-sl-tools.root/bin/cardano-wallet-hs2purs
 
+# Generate daedalus-bridge
 pushd daedalus
   nix-shell --run "npm install && npm run build:prod"
   echo $TRAVIS_BUILD_NUMBER > build-id
   cp ../log-config-prod.yaml .
 popd
 
+# Generate explorer frontend
+EXPLORER_EXECUTABLE=$(pwd)/cardano-sl-explorer-static.root/bin/cardano-explorer-hs2purs ./explorer/frontend/scripts/build.sh
```

Comment:
:confused: Wait. What? It's in a single line. And also, why did you put `$(pwd)` in it? Wasn't `.` enough?